jsonnet/kube-prometheus/components/node-exporter.libsonnet

```diff
@@ -28,6 +28,7 @@ local defaults = {
     ruleLabels: {},
     _config: {
       nodeExporterSelector: 'job="' + defaults.name + '"',
+      fsSpaceFillingUpWarningThreshold: 20,
```

Comment:
maybe we could have a comment explaining that this value is chosen to match with the default Kube GC settings?

Reply:
Thanks @simonpasquier. Added a comment to reason the adjustment.